File: sdkwork-ai-sdk-go/types/search_result.go
package types

// 检索结果
type SearchResult struct {
	Content string `json:"content"`
	Score float64 `json:"score"`
	Metadata map[string]interface{} `json:"metadata"`
	DocumentId string `json:"document_id"`
	ChunkId string `json:"chunk_id"`
}

File: sdkwork-ai-sdk-csharp/Models/SearchResult.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Ai.Models
{
    public class SearchResult
    {
        public string? Content { get; set; }
        public double? Score { get; set; }
        public Dictionary<string, object>? Metadata { get; set; }
        public string? DocumentId { get; set; }
        public string? ChunkId { get; set; }
    }
}
